### DISCUSSION TITLE
Sponsorship / GitHub Sponsors

### DISCUSSION THREAD

**Discussion by adamkdean** [nginx-proxy/nginx-proxy#2107] [General]
This project, along with acme-companion, is fantastic, and saves myself and I'm sure many others a lot of time and effort. I've used them both for years, and it'd be nice to be able to give something back.

Have the team considered enrolling in GitHub Sponsors?

**Comment by O5ten** [comment]
I concur with above speaker. It would be wonderful to be able to give something back.
